pySDC/tests/test_sweepers/test_MPI_sweeper.py

```diff
@@ -152,7 +152,7 @@ def test_sweeper(num_nodes, quad_type, residual_type, imex, initGuess, launch=Tr
     individual_test(num_nodes, quad_type, residual_type, imex, initGuess, useNCCL=False, launch=launch)
 
 
-@pytest.mark.cupy
+# @pytest.mark.cupy
```

Comment:
I think the cleanest way to implement this is to add a [skip decorator]. 
```
@pytest.mark.cupy
@pytest.mark.skip(reason="We haven\'t figured out how to run tests on the cluster with multiple processes yet.")
...
```

Reply:
Yes, correct. I just forgot it. 🤦‍♂️